src/results/parse.jl

```diff
@@ -352,6 +352,14 @@ function save_updated_gen_table(config, data)
         return row.pcap_inv_sim
     end
 
+    #update pcap_plant_avg, pcap_hist and pgen to -1 instead of -Inf to be excel compatible 
+    for row in eachrow(gen_tmp)
```

Comment:
Hmm, I wonder if it makes more sense to do this on the input side? I.e. so that -Inf is never introduced in the first place?  Let me know what you think

Reply:
We could do this, I made an e4st-input-processing issue about it, but going to merge this for now anyways.